evap/student/templates/student_vote.html

```diff
@@ -22,8 +22,8 @@
     {% if evaluation.is_midterm_evaluation %}
         <div class="callout callout-warning">{% trans 'The results of this evaluation will be published while the course is still running. This means that the contributors will receive this feedback before the final grades for the course have been published.' %}</div>
     {% endif %}
-    {% if evaluation_ends_soon %}
-        <div class="callout callout-warning">{% blocktrans %}The evaluation period will end in {{ hours_left_for_evaluation }} hours and {{ minutes_left_for_evaluation }} minutes ({{ vote_end_datetime }}). Your evaluation will only be accepted if you send the completed questionnaire before this deadline ends.{% endblocktrans %}</div>
+    {% if evaluation.evaluation_ends_soon %}
```

Comment:
It feels a bit weird to double the word evaluation here. Maybe: `evaluation.is_in_extra_evalution_period` (I couldn’t think of a smart name for this time period for the time period between displayed deadline and actual deadline. I proposed elsewhere to display “ends soon” for the displayed deadline too, if you don’t want that, `evaluation.does_end_soon` would suffice.)

Reply:
`evaluation.ends_soon`